modules/jt-migrating-from-jenkins-plugins-to-tekton-hub-tasks.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-migrating-from-jenkins-plugins-to-tekton-hub-tasks_{context}"]
+= Migrating from Jenkins plugins to Tekton Hub tasks
+
+toc::[]
+
+
+In Jenkins, there are many link:https://plugins.jenkinsci.org[plugins] that help you extend its capability. To achieve similar extensibility in Tekton, you can use many readily available tasks from the link:https://hub.tekton.dev[Tekton Hub].
```

Comment:
Optional rephrase to cut down on word count:
`To achieve similar extensiblity in Tekton, use any of the readily available tasks from the Tekton Hub.`